File: src/main/java/de/letsdev/products/dico/scanner/backend/configuration/MvcConfig.java
package de.letsdev.products.dico.scanner.backend.configuration;

import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.PropertySource;
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
import org.springframework.web.servlet.config.annotation.ViewControllerRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

@Configuration
@PropertySource(value = "classpath:/application.properties", encoding="UTF-8")
public class MvcConfig implements WebMvcConfigurer {

    @Override
    public void addViewControllers(ViewControllerRegistry registry) {

        registry.addViewController("/index").setViewName("index");
        registry.addViewController("/").setViewName("index");
    }

    @Override
    public void addResourceHandlers(ResourceHandlerRegistry registry) {

        registry.addResourceHandler("file://" + System.getProperty("user.dir") + "/project-home/")
                .addResourceLocations("file://" + System.getProperty("user.dir") + "/project-home/");
    }
}

File: src/main/java/de/letsdev/products/dico/scanner/backend/MvcConfig.kt
package de.letsdev.products.dico.scanner.backend

import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry
import org.springframework.web.servlet.config.annotation.ViewControllerRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
/* TODO Convert to Java!*/
@Configuration
class MvcConfig : WebMvcConfigurer {

    override fun addViewControllers(registry: ViewControllerRegistry) {
        registry.addViewController("/index").setViewName("index")
        registry.addViewController("/").setViewName("index")
    }

    override fun addResourceHandlers(registry: ResourceHandlerRegistry) {
        registry.addResourceHandler("file://" + System.getProperty("user.dir") + "/project-home/").addResourceLocations("file://" + System.getProperty("user.dir") + "/project-home/")
    }
}